src/main/play-doc/operation/Install.md

```diff
@@ -729,6 +784,28 @@ SSH to the bastion host. Once logged in, export the `CONDUCTR_IP` environment va
 
 You are now ready to [[deploy bundles|DeployingBundlesOps]] into ConductR.
 
+
+## Loading and Running ConductR-HAProxy Bundle
+
+ConductR-HAProxy bundle will expose services provided by the bundle so it can be accessed from outside cluster. ConductR-HAProxy will ensure the HAProxy configuration is kept update based on the bundles which are running.
+
+First we need to obtain the `.tgz` ConductR installations package.
+
+> In order to obtain the installations of ConductR then please [contact our sales department](https://www.lightbend.com/company/contact). To evaluate ConductR in general then [please visit our product page](http://www.lightbend.com/products/conductr) which provides instructions on getting started. Otherwise if you are looking to use ConductR for free from a development perspective then please [head over to our developer section](DevQuickStart).
+
+Once the ConductR `.tgz` is obtained, extract ConductR HAProxy bundle from the package. The ConductR HAProxy bundle can be found within the ConductR package under  `conductr-{version}/extra/conductr-haproxy-{version}-{digest}.zip`.
```

Comment:
@huntc and @edwardcallahan - obtaining ConductR HAProxy bundle is now done through downloading Conductr `.tgz` from download page.